Solutions/Infoblox Cloud Data Connector/Data Connectors/InfobloxCloudDataConnector.json

```diff
@@ -75,6 +84,16 @@
         ]
     },
     "instructionSteps": [
+        {
+            "title": "",
+            "description": ">**IMPORTANT:** This data connector depends on a parser based on a Kusto Function to work as expected called [**InfobloxCDC**](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Infoblox%20Cloud%20Data%20Connector/Parsers/InfobloxCDC.txt) which is deployed with the Azure Sentinel Solution.",

```

Comment:
Please update parser short link **https://aka.ms/sentinel-InfobloxCloudDataConnector-parser** instead of https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Infoblox%20Cloud%20Data%20Connector/Parsers/InfobloxCDC.txt. Thanks

Reply:
@NikTripathi ah dang!! missed one. updated. thanks!